Library/Homebrew/cask/lib/hbc/version.rb

```diff
@@ -1,13 +1,10 @@
-HBC_VERSION = "0.60.0".freeze
+HBC_VERSION = "0.60.1".freeze
```

Comment:
Is it worth maintaining a separate version for the two projects? Longer-term at least this should go away.